composer.json

```diff
@@ -39,7 +39,8 @@
 		}
 	],
 	"require": {
-		"php": ">=5.3.3",
+		"php": ">=5.4",
+		"phpbb/phpbb": ">=3.2.0@dev",
```

Comment:
Libraries should not have .lock files. 
About your initial question: no because it will be installed/required from phpbb. 

After that, should we set the limit to x or y, MichaelC said y las time (because for now we can't ensure that 3.3 will not break anything as the core doesn't really follow sever for now. And anyway I can ensure you that 3.3 will break something,  and of the next major version is called 4.0 then 3.3 will never exists. )